src/lib/sensor_calibration/Magnetometer.hpp

```diff
@@ -103,7 +116,11 @@ class Magnetometer
 private:
 	Rotation _rotation_enum{ROTATION_NONE};
 
+	/**
+	 * @brief 3 x 3 Rotation matrix that translates from sensor frame (XYZ) to vehicle body frame (FRD)
+	 */
 	matrix::Dcmf _rotation;
```

Comment:
@dagar I in general notice that the frame rotation matrices / biases don't have a clear description, so it's unclear when & where the bias should be applied (before or after the rotation?).

As you mentioned in [this comment](https://github.com/PX4/PX4-Autopilot/pull/20543#issue-1434820706), I think partial solution would be to be more verbose in naming / commenting on the bias / rotation matrices on exactly what they are supposed to do.

Thoughts?

Reply:
> be more verbose in naming / commenting on the bias / rotation matrices

both. please.

@junwoo091400  since you specified this rotation at least in a comment, i don't want to drag out this PR any longer. perhaps you could make an issue though, and we can try to get a discussion started on defining a convention? i.e. what specific naming for rotation matrices and quaternions should we try to be consistent with throughout the code base?

Reply:
Issue created: https://github.com/PX4/PX4-Autopilot/issues/20639